ballerina-test/src/test/resources/openapi/expected/filtered_tags.bal

```diff
@@ -1,16 +1,17 @@
-// AUTO-GENERATED FILE.
-// This file is auto-generated by the Ballerina OpenAPI tool.
-import ballerina/http;
+ // AUTO-GENERATED FILE.
+    // This file is auto-generated by the Ballerina OpenAPI tool.
 
-listener http:Listener ep0 = new (80, config = {host: "petstore.openapi.io"});
+    import ballerina/http;
```

Comment:
Why this indentation added to this file? Is it an auto-generated?

Reply:
It was a formatting issue. will fix.